healthcheck/settings/base.py

```diff
@@ -138,3 +122,20 @@
 # this might be unneccessary if intented usage is international phone numbers
 PHONENUMBER_DB_FORMAT = "NATIONAL"
 PHONENUMBER_DEFAULT_REGION = "ZA"
+
+# CELERY SETTINGS
+CELERY_BROKER_URL = env.str("CELERY_BROKER_URL", "redis://localhost:6379/0")
+CELERY_RESULT_BACKEND = env.str("CELERY_RESULT_BACKEND", "redis://localhost:6379/0")
+CELERY_ACCEPT_CONTENT = ["application/json"]
+CELERY_TASK_SERIALIZER = "json"
+CELERY_RESULT_SERIALIZER = "json"
+
+# TEMPORARY SETUP - REPLACE WITH @PERIODIC IN PRODUCTION
```

Comment:
This is what will be running in production, we can't change files to be different in production.

Does this mean that you'll change this before merging?

Reply:
Yes, I will. I'm going to fix all the other issues and change this to periodic task

Reply:
Changed these to be configurable, made the task periodic.